Automattic/wp-calypso
# Style Selection: Add theme style variation icons

#### Proposed Changes

This PR adds available style variations to theme meta. See design spec: 7l53h5fxAUNjVRBQ82YFwn-fi-238%3A41198
Clicking any variation icon should open the full-screen theme preview, although it's worth mentioning that the entire section is clickable since it's within a button element.

![Screen Shot 2022-08-24 at 5 22 21 PM](https://user-images.githubusercontent.com/797888/186381876-83c12217-85b4-4a7c-8ebb-cea1762df77f.png)

Note that this PR depends on D86264-code, and because of that the API payload might change.

#### Testing Instructions



* Head to the unified design picker `/setup/designSetup?siteSlug=${site_slug}`.
* Ensure that the style variation icons are as per design spec.
* Ensure that clicking on any style variation icon should open the full-screen theme preview.

#### Pre-merge Checklist






Related to #66734
